app/scripts/contentscript.js

```diff
@@ -124,23 +131,28 @@ const requestMetrics = () => {
  * Init all plugin elements
  */
 const init = () => {
-    if(window.location.hash ==  '#hubcare'){
-        cleanPageContent()
+    console.log('oooooooooooooooooirrr ==' + chaves)
+    if(chaves=="abrir"){
+        if(window.location.hash ==  '#hubcare'){
+            cleanPageContent()
+        }
+        insertActivityIndicator()
+        insertButton()
+        requestMetrics()
+        document.getElementById('hubcare-button').addEventListener("click", function() {
+            hubcarePage()
+        }, false);
+
     }
-    insertActivityIndicator()
-    insertButton()
-    requestMetrics()
-    document.getElementById('hubcare-button').addEventListener("click", function() {
-        hubcarePage()
-    }, false);
 }
 
 const hubcarePage = () => {
     cleanPageContent()
     createCommitChart()
 }
-init()
+
+//init()
```

Comment:
The hubcare is not starting, it seems that you commented the init

Reply:
Was created new way to resolve this

Reply:
seems solved